### ISSUE TITLE
directives should not be parsed as filters

### ISSUE THREAD

**Issue by armano2** [mysticatea/vue-eslint-parser#34]
After last version binary operator `|` in directives is parsed as Filter instead of `BinaryExpression`

```vue
<template>
    <input v-html="foo | bar">
</template>
<script>
export default {
  data() {
    return { foo: 200, bar: 5 }
  }
}
</script>
```

**Comment by mysticatea** [comment]
Thank you for the report!

Indeed, it's a bug.
